Datadog.Trace.sln

```diff
@@ -196,42 +196,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Trace.ClrProfiler.M
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AssemblyLoad.FileNotFoundException", "test\test-applications\regression\AssemblyLoad.FileNotFoundException\AssemblyLoad.FileNotFoundException.csproj", "{1A5E9F40-F3A5-4B59-9898-3DCD65C459C3}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docker", "docker", "{6ABAD006-E206-488E-ACA1-1AA73F9B5146}"
-	ProjectSection(SolutionItems) = preProject
-		build\docker\build.arm64.sh = build\docker\build.arm64.sh
-		build\docker\build.sh = build\docker\build.sh
-		build\docker\Datadog.Trace.ClrProfiler.IntegrationTests.arm64.sh = build\docker\Datadog.Trace.ClrProfiler.IntegrationTests.arm64.sh
-		build\docker\Datadog.Trace.ClrProfiler.IntegrationTests.sh = build\docker\Datadog.Trace.ClrProfiler.IntegrationTests.sh
-		build\docker\Datadog.Trace.ClrProfiler.Native.sh = build\docker\Datadog.Trace.ClrProfiler.Native.sh
-		build\docker\dotnet.alpine.core21.dockerfile = build\docker\dotnet.alpine.core21.dockerfile
-		build\docker\dotnet.alpine.core30.dockerfile = build\docker\dotnet.alpine.core30.dockerfile
-		build\docker\dotnet.alpine.core31.dockerfile = build\docker\dotnet.alpine.core31.dockerfile
-		build\docker\dotnet.alpine.core50.dockerfile = build\docker\dotnet.alpine.core50.dockerfile
-		build\docker\dotnet.arm64.core50.dockerfile = build\docker\dotnet.arm64.core50.dockerfile
-		build\docker\dotnet.arm64.dockerfile = build\docker\dotnet.arm64.dockerfile
-		build\docker\dotnet.dockerfile = build\docker\dotnet.dockerfile
-		build\docker\linux-build.bash = build\docker\linux-build.bash
-		build\docker\linux-build.dockerfile = build\docker\linux-build.dockerfile
-		build\docker\native.alpine.dockerfile = build\docker\native.alpine.dockerfile
-		build\docker\native.dockerfile = build\docker\native.dockerfile
-		build\docker\package.alpine.dockerfile = build\docker\package.alpine.dockerfile
-		build\docker\package.dockerfile = build\docker\package.dockerfile
-		build\docker\package.sh = build\docker\package.sh
-		build\docker\with-profiler-logs.bash = build\docker\with-profiler-logs.bash
-		build\docker\with-profiler.bash = build\docker\with-profiler.bash
-	EndProjectSection
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".azure-pipelines", ".azure-pipelines", "{C52D6695-4E05-4930-88F8-0EFF8056A967}"
-	ProjectSection(SolutionItems) = preProject
-		.azure-pipelines\benchmarks.yml = .azure-pipelines\benchmarks.yml
```

Comment:
Should we add `ultimate-pipeline.yml` to the solution?

Reply:
It's already part of the solution as it's automatically included by Nuke in the build project🙂 

![image](https://user-images.githubusercontent.com/18755388/124615261-0153c200-de6d-11eb-8c84-afc0c950ad81.png)

Reply:
Awesome, thanks!